core/src/main/java/org/infinispan/commands/functional/AbstractWriteManyCommand.java

```diff
@@ -88,7 +88,7 @@ public boolean isConditional() {
 
    @Override
    public boolean canBlock() {
-      return true;
+      return false;
```

Comment:
@danberindei @wburns WDYT to adding a default implemntation to `ReplicableCommand` that returns false? It will remove a fair bit of boilerplate.

Reply:
I had debated about it, but didn't try it. I can see how it feels. I was debating about even deprecating the method as well.

Reply:
Deprecating it sounds good. At the very least it gives us the option to remove it in 12 even if we're not ready to do that.